lib/perl/Genome/Model/SingleSampleGenotype/Command/QualityControl.pm

```diff
@@ -35,6 +35,7 @@ sub _params_for_command {
     my @params = (
         alignment_result => $build->merged_alignment_result,
         config_name => $build->model->qc_config,
+        qc_genotype_vcf_file => $build->model->qc_genotype_vcf_file,
```

Comment:
Inputs should be obtained from the `$build` directly rather than going back to the model.